handler/PostMessageSentHandler.ts

```diff
@@ -100,8 +100,13 @@ export class PostMessageSentHandler {
                                 this.modify,
                                 { text: serviceUnavailable ? serviceUnavailable : DefaultMessage.DEFAULT_DialogflowServiceUnavailableMessage });
 
-            updateRoomCustomFields(rid, { isChatBotFunctional: false }, this.read, this.modify);
             const targetDepartment: string = await getAppSettingValue(this.read, AppSetting.FallbackTargetDepartment);
+            if (!targetDepartment) {
+                console.error(Logs.EMPTY_HANDOVER_DEPARTMENT);
```

Comment:
@ItzzRitik @Shailesh351 I have a question about this logging please.  When using this 'console.error' logging, the log points are showing up in stdout....  but above at line 94, using 'this.app.getLogger().error', I am not seeing that logpoint anywhere.  Is there a bug in our app.Logger?

Reply:
actually I do see the log point in the RC/Admin/Apps/Dialogflow/'view logs' UI that can be reached by clicking the three dot menu......  but it does not show up anywhere in stdout....  in general, we need to count on stdout for all of our error logs because we view and alert on them in Kibana.

Reply:
@ear-dev `this.app.getLogger()` is only meant for App specific logging in `RC/Admin/Apps/Dialogflow/'view logs' UI`. It does not show up in stdout.

Only console logs we are adding will be visible in stdout

Reply:
ok, gottit.... @ItzzRitik can you add console error logging to that line at 94 please?  We need to see any http errors in kibana....  is there a way to log in both places please?

Reply:
Sure Eric, I'll do it